xtos/include/sof/lib/perf_cnt.h

```diff
@@ -25,6 +25,8 @@ struct perf_cnt_data {
 	uint32_t cpu_delta_peak;
 	uint32_t cpu_delta_sum;
 	uint32_t sample_cnt;
+	/* indicate which period have peak mcps in current module */
```

Comment:
Indicate which period or sample when the peak occurred? To keep it aligned with existing perf_cnt code I would suggest  to use 'sample' word or rename sample_cnt to period_cnt.

Reply:
@btian1 its probably worth adding a comment to each member in this structure so that its easy to follow (since the structure is now getting busier).

Reply:
ok